test/Microsoft.AspNet.Razor.Test/Parser/CSharp/CSharpDirectivesTest.cs

```diff
@@ -12,29 +12,16 @@ namespace Microsoft.AspNet.Razor.Test.Parser.CSharp
     public class CSharpDirectivesTest : CsHtmlCodeParserTestBase
     {
         [Fact]
-        public void TagHelperPrefixDirective_NoValueSucceeds()
-        {
-            ParseBlockTest("@tagHelperPrefix \"\"",
-                new DirectiveBlock(
-                    Factory.CodeTransition(),
-                    Factory
-                        .MetaCode(SyntaxConstants.CSharp.TagHelperPrefixKeyword + " ")
-                        .Accepts(AcceptedCharacters.None),
-                    Factory.Code("\"\"")
-                        .AsTagHelperPrefixDirective(string.Empty)));
-        }
-
-        [Fact]
         public void TagHelperPrefixDirective_Succeeds()
         {
-            ParseBlockTest("@tagHelperPrefix \"Foo\"",
```

Comment:
:ok: my interest was primarily in the second question and that experience seems reasonable for those moving from prior versions.